python/rmm/rmm/tests/test_rmm.py

```diff
@@ -524,7 +524,7 @@ def test_reinitialize_initial_pool_size_gt_max():
     with pytest.raises(RuntimeError) as e:
         rmm.reinitialize(
             pool_allocator=True,
-            initial_pool_size=1 << 11,
+            initial_pool_size="2KiB",
             maximum_pool_size=1 << 10,
         )
     assert "Initial pool size exceeds the maximum pool size" in str(e.value)
```

Comment:
*nit*: Can we please test two further things:

1. The case where we still pass an integer as the pool size.
2. An invalid specification, to ensure it raises `ValueError`.

Thanks!

Reply:
I reverted this change and added a test for passing a string argument

Reply:
Thanks!